app/services/induction/send_notification_emails.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+module Induction
+  # This service assumes that all transfers are requested by the incoming school for now. There
+  # are three paths here:
+  #
+  # 1) Moving schools but lead provider is same at both schools:
+  #
+  #   a. Send email to existing lead provider notifying them that an internal
+  #      transfer is happening.
+  #   b. Send email to participant to notify them.
+  #
+  # 2) Moving to target schools lead provider and programme:
+  #
+  #   a. Send email to incoming lead provider.
+  #   b. Send email to outgoing lead provider, requesting that they withdraw them.
+  #   c. Send email to participant to notify them.
+  #
+  # 3) Moving to target school, but continuing with current lead provider.
+  #
+  #   a. Send email to current lead provider, notifying them to expect a new school.
+  #   b. Send email to participant to notify them.
+  #
+  class SendNotificationEmails < BaseService
```

Comment:
Could this be named something like `SendTransferNotificationEmails` to make it a bit clearer?